integration-tests/mysql-client-tests/mysql-client-tests.bats

```diff
@@ -100,6 +100,17 @@ cmake ..
     ruby $BATS_TEST_DIRNAME/ruby/ruby-mysql-test.rb $USER $PORT $REPO_NAME
 }
 
+@test "elixir myxql test" {
+    cd $BATS_TEST_DIRNAME/elixir/
+    # install some mix dependencies
```

Comment:
can we do this in the dockerfile with the rest of the deps?

Reply:
I think elixir expects dependencies to be local so I have to leave it like this :/ Tried the other way around and it didn't work